### PR TITLE
Issue #435: use php-cs-fixer for code consistency

### PR DESCRIPTION
**Pull request by pascalchevrel**

### CODE REVIEW

**Comment by pascalchevrel** [comment]
The build failing is actually because it now runs php-cs-fixer and tells us all the files that don't follow our coding standards, good news, travis integration works ;)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,6 +22,15 @@ A few key points to take into account:
 * Add a blank line before return statements, unless the return is alone inside a statement-group (like an if statement);
 * Use braces to indicate control structure body regardless of the number of statements it contains;
 
+Please use [php-cs-fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) before submitting your code for review. This tool will reformat all of your code to follow our coding standards.
+
+The rules we follow are formalized in a .php_cs file at the root of the project. So as to use our ruleset, you have to:
```
Comment (flodolo) [inline CONTRIBUTING.md]: In order to use our ruleset

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -22,6 +22,15 @@ A few key points to take into account:
...
+1. be at the root of the project
```
Comment (flodolo) [inline CONTRIBUTING.md]: You need to leave an empty line before this one to make this a list. Nit: start with uppercase, add period at the end.

**Comment by flodolo** [comment]
Really nice :+1: 
We should start using it in other projects as well, and recycle the ruleset

**Pull request merged**